docs/approvals/service-now/index.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Service Now Integration
+description: Approvals, approvals, approvals
+position: 10
+---
+:::warning
+The ServiceNow Integration feature is only available as an EAP for version 2022.2.5556 or later and it requires a feature license to be able to use it.
+:::
+
+## Overview
+
+The Octopus Deploy/ServiceNow integration allows users to block the execution of specifically configured deployments unless they have a corresponding approved ServiceNow **Change Request** (CR).
+
+For configured project and environment combinations, Octopus Deploy will 
+
+To enable this behavior, both systems must be configured before deployments can be managed.
+
+| Project | Environment Change Controlled | Environment **Not** Change Controlled| 
+| --------|--|-- |
+| Change Controlled | _Approval Required_  | No Approval Required | 
+| **Not** Change Controlled | No Approval Required | No Approval Required |
+
+## Getting started
+
+The Service Now integration requires Octopus 2022.2.5556 or later, and an Octopus license with the Service Now Integration feature enabled.
+
+To get started:
+1. Configure ServiceNow
+1. Obtain a ServiceNow feature license for your Octopus instance 
+1. Configure a connection from Octopus to ServiceNow
+1. Configure which deployments require an approved CR
+
+### Configuring Service Now
+
+:::hint
+The instructions in this section will require a ServiceNow Administrator.
+:::
+
+The Octopus Deploy / ServiceNow integration requires security configuration in your target ServiceNow instance. 
```

Comment:
does this need an introduction?
Two piece of work must be configured in ServiceNow:
1. Create ServiceAccount for use by Octopus
2. Create an OAuth endpoint ...
Why do we need the OAuth? Is it safer, or is it simply mandatory?

Reply:
🤷 

does this need more explanation given https://octopusdeploy.slack.com/archives/C0K9DNQG5/p1653433973150239